scheduler/server/core/storage.go

```diff
@@ -16,17 +16,15 @@ package core
 import (
 	"encoding/json"
 	"fmt"
-	"math"
-	"path"
-	"strconv"
-	"strings"
-	"sync/atomic"
-
 	"github.com/gogo/protobuf/proto"
 	"github.com/pingcap-incubator/tinykv/proto/pkg/metapb"
 	"github.com/pingcap-incubator/tinykv/scheduler/server/kv"
 	"github.com/pkg/errors"
 	"go.etcd.io/etcd/clientv3"
+	"math"
```

Comment:
please keep the order as before

Reply:
Oops, this is a mistake of my IDE :cry:. Should we add `goimports -w -l ./` into Makefile to automatically group imports?

Reply:
you can add it in format check

Reply:
OK! I will do it in another PR.